bot/seasons/evergreen/nationday.py

```diff
@@ -0,0 +1,162 @@
+            return [(err_msg, white_flag)]
+        # Create pages
+        countries = list(countries.split(','))
+        pages = []
+        for country in countries:
+            page, img = await self.get_country_info(self.http_session, country)
+            pages.append((page, img))
+        return pages
+
+    async def get_country_info(self, client: ClientSession, country: str) -> Tuple[str, str]:
+        """
+        Create country information page using RESTCountries API.
+
+        Return page and flag image.
+        """
+        try:
+            iso_code = self.iso_codes[country]
+            if iso_code is None:
+                not_available_msg = f"Could not find information on this country."
+                logger.warning(not_available_msg)
+                return not_available_msg, white_flag
+            async with client.get(url+iso_code) as resp:
+                info = await resp.json()
+
+            country_info = ""
+            country_info += f"**Name**: {info['name']}\n\n"
+            country_info += f"**Region**: {info['region']}\n\n"
+            country_info += f"**Capital**: {info['capital']}\n\n"
+            country_info += f"**Population**: {info['population']:,}\n\n"
+            country_info += f"**Currency**: {info['currencies'][0]['name']} ({info['currencies'][0]['symbol']})\n"
+            flag = flag_url.format(country_code=iso_code)
+
+        except Exception as e:
+            msg = f"Error! {e}"
+            logger.warning(e)
+            return msg, white_flag
+
+        return country_info, flag
+
+    @command(name='nationday')
+    async def nationday(self, ctx: Context, param: str = "") -> None:
+        """
+        \U0001F30F NationDay Help.
+
+        Enter a country name to get independence day of that country along with some basic information on the country.
+        Enter "today" to get all countries whose independence day is the current day, along with information.
+        Usage:
+        -> .nationday today
+        -> .nationday [country] (use "" for countries with space in the name)
+        Examples:
+        -> .nationday india
+        -> .nationday colombia
+        -> .nationday today
+        -> .nationday "United Arab Emirates"
+        -> .nationday "Ivory Coast"
+        """
+        param = param.lower()
+
+        if param == 'today':
+            pages = await self.country_today()
+            embed = discord.Embed(
+                title='Countries that have their independence days today'
+            ).set_footer(text='Powered by the RESTCountries API.')
+            await ImagePaginator.paginate(pages, ctx, embed)
+
+        # Check if country is present
+        elif param in self.iso_codes.keys():
+            page, date = await self.get_specific_country(param)
+            if page[0][0] is not None:
+                param_list = list(param.split(' '))
+                param_list = [x.capitalize() for x in param_list]
+                param = ' '.join(param_list)
```

Comment:
Though this should have a comment explaining what's happening and use more explicit variable names, we can accomplish capitalization of the country name directly with [`str.title()`](https://docs.python.org/3/library/stdtypes.html#str.title)